packages/@webex/plugin-meetings/src/media/index.ts

```diff
@@ -383,20 +374,21 @@ Media.toggleStream = () => {};
  * @param {LocalStream} stream A local stream
  * @returns {null}
  */
-Media.stopStreams = (stream: LocalStream) => {
+Media.stopStream = (stream: LocalStream) => {
   if (!stream) {
     return Promise.resolve();
   }
 
   return Promise.resolve().then(() => {
-    if (stream && stream.stop) {
-      try {
-        stream.stop();
-      } catch (e) {
-        LoggerProxy.logger.error(
-          `Media:index#stopTracks --> Unable to stop the track with state ${stream.outputTrack.readyState}, error: ${e}`
-        );
-      }
+    try {
+      stream.stop();
+    } catch (e) {
+      LoggerProxy.logger.error(
+        `Media:index#stopTracks --> Unable to stop the output stream with state ${stream.outputTrack.readyState}, error: ${e}`
+      );
+      LoggerProxy.logger.error(
+        `Media:index#stopTracks --> Unable to stop the input stream with state ${stream.inputTrack.readyState}, error: ${e}`
```

Comment:
very minor, but it would be nicer for the 2 logs to say which one is for inputTrack and which one for outputTrack, or it could be even all in a single log

Reply:
It is already separated out as `input stream` and `output stream` but will make it one log yes.

Reply:
ah, sorry, yeah I didn't notice that. Thanks for combining it into a single log